### PR TITLE
feat: added 'Using a multisig account on Astar Portal' page

### PR DESCRIPTION
**Pull request by impelcrypto**
* feat: added 'Using a multisig account on Astar Portal' page
  * preview: https://astar-docs--pr374-feat-add-multisig-po-k2earvl0.web.app/docs/user-guides/using-a-multisig-account-on-astar-portal 
* refactor: polished some sentences (adding '.' and removing empty spaces)

* Staging URL (portal): https://astar-apps--pr834-feat-polkasafe-integ-b1tcwfpc.web.app/

### CODE REVIEW

File: docs/user-guides/using-a-multisig-account-on-astar-portal.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+This guide will assist you in utilizing a multisig account on Astar Portal via PolkaSafe.
+
+:::danger
+At the time of this release, PolkaSafe only supports Astar Network.
+:::
+
+
+## **Prerequisites**
+
+- Install a substrate wallet extension (such as Polkadot.js, SubWallet, or Talisman) on your browser.
```
Comment (midegdugarova) [inline docs/user-guides/using-a-multisig-account-on-astar-portal.md:16]:
```suggestion
- Install a Substrate wallet extension (such as Polkadot.js, SubWallet, or Talisman) on your browser.
```

File: docs/user-guides/using-a-multisig-account-on-astar-portal.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+1. **Enhanced Security:** The need for more keys reduces the risk of unauthorized access.
+2. **Reduced Risk:** You won't lose access to the account even if you lose a single key.
+3. **Access Control:** Multisig allows multiple users to share control over an account.
+4. **Flexibility:** You can customize the number of required signatures for transactions.
+5. **Dispute Resolution:** Multisig prevents unilateral decisions by requiring majority or total agreement.
+6. **Auditability:** Multisig accounts provide a clear record of approved transactions.
+
+## Creating a Multisig Account on PolkaSafe
+
+PolkaSafe integrates with Polkadot's multisig solution, offering a seamless user experience and robust security features for managing assets in the Polkadot ecosystem. You can use the multisig accounts created on PolkaSafe with the Astar Portal.
```
Comment (midegdugarova) [inline docs/user-guides/using-a-multisig-account-on-astar-portal.md:32]:
```suggestion
PolkaSafe integrates with Polkadot's multisig solution, offering seamless user experience and robust security features for managing assets in the Polkadot ecosystem. You can use the multisig accounts created on PolkaSafe with the Astar Portal.
```

File: docs/user-guides/using-a-multisig-account-on-astar-portal.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+1. Visit [PolkaSafe](https://app.polkasafe.xyz/).
+2. Select the Astar network and click 'Connect Wallet.'
```
Comment (midegdugarova) [inline docs/user-guides/using-a-multisig-account-on-astar-portal.md:35]:
```suggestion
2. Select  Astar network and click 'Connect Wallet.'
```

**Comment by impelcrypto** [comment]
@midegdugarova Thanks, I'll merge this PR after next version up in our portal

**Review by gluneau — APPROVED** [review]
LGTM

**Pull request merged**